Assets/Scripts/Core/AnimalManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Anivision.PlayerInteraction;
+using Anivision.Vision;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Anivision.Core
```

Comment:
Ok I'm going to stop commenting on this sorry

Reply:
Hahaha, animal manager is in charge of switching animal prefabs, and animal prefabs contains info on movement and vision, not just vision. So that's the reasoning there. If you have other ideas for organization though I'm all ears, lmk.

Reply:
OK, yeah that makes sense, I should have thought about it more